app/src/main/java/np/com/naxa/factsnepal/splashscreen/SplashScreenActivity.java

```diff
@@ -23,11 +24,19 @@ protected void onCreate(Bundle savedInstanceState) {
             public void run() {
                 //Do something after 100ms
                 SharedPreferenceUtils sharedPreferenceUtils = new SharedPreferenceUtils(SplashScreenActivity.this);
-                if (sharedPreferenceUtils.getBoolanValue(LoginActivity.KEY_IS_USER_LOGGED_IN, false)) {
```

Comment:
Also please maintain  shared pref keys in `Constant` class inside ` SharedPrefKey class `
```
public class Constant {
    public static final String KEY_OBJECT = "object";
    public static final String KEY_EXTRA_OBJECT = "extra_object";
    public static final String EXTRA_IMAGE = "extra_image";
    public static String RANDOM_PHOTO = "https://picsum.photos/1000/600/?random";
    public static String FACT_PHOTO = "https://scontent.fktm8-1.fna.fbcdn.net/v/t1.0-9/52638797_2396505700373002_4565827759254798336_n.png?_nc_cat=105&_nc_ht=scontent.fktm8-1.fna&oh=7bbb8ead8c153edfe8e2f1af17f8eef2&oe=5D134EF8";

    public class SharedPrefKey {
        public static final String SELECTED_CATEGORIES = "selected_categories";


    }
}
```